package.json

```diff
@@ -51,7 +51,7 @@
     "src"
   ],
   "dependencies": {
-    "uswds": "^2.0.0-beta"
+    "uswds": "^2.0.0-beta.3"
```

Comment:
I had a build issue when I tried to use beta.3 in NGWMN. I'm wondering what the difference is. I should have opened a ticket. My bad.